src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -164,11 +167,25 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
     setWithdrawAllSelected(withdrawAll);
   };
 
-  const onChangeDate = (id: string, value?: any) => {
-    const date = new Date(value);
-    if (isNaN(date.getTime()) || isInTheFuture(date)) {
-      return;
+  const validateDate = (value?: any) => {
+    if (!value) {
+      setIndividualError('date', strings.REQUIRED_FIELD);
+      return false;
+    } else if (isNaN(value.getTime())) {
```

Comment:
Does `getTime` give the milliseconds since epoch number?

Reply:
Exactly !